terraform/modules/hub/variables.tf

```diff
@@ -83,6 +83,11 @@ variable "matomo_salt" {
   description = "Secret salt for matomo cookies and fingerprinting hashes"
 }
 
+variable "matomo_db_instance_class" {
+  description = "EC2 instance class for matomo"
+  default     = "db.m4.large"
```

Comment:
can the default be an `m5` family? I know the old one is `m4` but if we're moving house we shouldn't take our rubbish with us 😄

Reply:
m5-ified